.gitignore

```diff
@@ -146,5 +146,9 @@ log/
 # Ignore olm repository
 resources/olm
 
+# Ignore sqlcipher repository
+resources/sqlcipher
```

Comment:
Are both present?

Reply:
It's an artifact of me trying to build dependencies by myself. I switched to packages and pip modules as it was in the end more reliable than cloning.